stdlib/http/cookiejar.pyi

```diff
@@ -1,14 +1,20 @@
 import sys
 from _typeshed import StrPath
 from http.client import HTTPResponse
-from typing import Dict, Iterable, Iterator, Optional, Sequence, Tuple, TypeVar, Union, overload
+from typing import Dict, Iterable, Iterator, Optional, Pattern, Sequence, Tuple, TypeVar, Union, overload
 from urllib.request import Request
 
 _T = TypeVar("_T")
 
 class LoadError(OSError): ...
 
 class CookieJar(Iterable[Cookie]):
+    non_word_re: Pattern[str] = ...  # undocumented
+    quote_re: Pattern[str] = ...  # undocumented
+    strict_domain_re: Pattern[str] = ...  # undocumented
+    domain_re: Pattern[str] = ...  # undocumented
+    dots_re: Pattern[str] = ...  # undocumented
+    magic_re: Pattern[str] = ...  # undocumented
```

Comment:
I think these should use `ClassVar`. They are defined in the class, not in `__init__`.

Reply:
thanks, didn't know about `ClassVar`, but makes totally sense here